db/seeds.rb

```diff
@@ -10,13 +10,14 @@
 Item.delete_all
 User.delete_all
 
-User.create(email: 'vasyll@interlink-ua.com', password: 'password')
+user = User.create(email: 'vasyll@interlink-ua.com', password: 'password')
 
 first_list = List.create(title: 'Explore Basecamp! — Some quick things to explore in Basecamp',
-  description: 'A nice way to get acclimated.'
+  description: 'A nice way to get acclimated.',
+  user_id: user.id
```

Comment:
@vlasiak Please take a look at [the suggestion of formatting](https://github.com/bbatsov/ruby-style-guide#no-double-indent) of many attributes.